### PR TITLE
.gitignore housekeeping

### PR DESCRIPTION
**Pull request by anewmanRH**
Exclude conf/ and all YAMLs

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit a5d44c746baffca47eb61c2694ead89319596b62:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: .gitignore [hunk 1]
```diff
@@ -66,3 +59,15 @@ notebooks/.ipynb_checkpoints
 cfme-qe-yamls
 rhci-qe-data
 
+# exclude conf
+conf/
```
Comment (mfalesni) [inline .gitignore]: I would rather say `conf/*.yaml` because we want the templates to be there and our config files always have `.yaml` extension ...

**Comment by dajoRH** [comment]
Lint report for commit 355df83debbef7bc5c20b03511c5a0485b28f53a:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit da6f3e45d9d2171d5db1efa5be0a67de4b6d3159:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 9db4b75fe421fe6b4dc1eb16344008f7dadac08a:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by psav** [comment]
This will need squashing out before being ready to be merged

**Comment by dajoRH** [comment]
Lint report for commit f0b5895ca58864990aceb56aa745b811c5398780:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 4302b639d19e0046a11e29e7375c2dcb61c427c2:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**